src/app/loans/create-loans-account/create-loans-account.component.ts

```
@@ -133,43 +159,58 @@ export class CreateLoansAccountComponent {
+        ...this.loansAccountTermsStep?.loansAccountTerms,
+        ...this.loansAccountChargesStep?.loansAccountCharges,
+        ...this.loansAccountTermsStep?.loanCollateral,
+        ...this.loansAccountTermsStep?.disbursementData
+      };
+    } else if (this.loanProductService.isWorkingCapital) {
+      return {
+        ...this.loansAccountDetailsStep.loansAccountDetails,
+        ...this.loansAccountTermsStep?.loansAccountTerms
+      };
+    }
   }
```

Biome (2.4.4)

[error] This getter should return a value.

(lint/suspicious/useGetterReturn)